Services/COPage/PC/Login/class.ilPCLoginPageElement.php

```diff
@@ -30,7 +30,6 @@ class ilPCLoginPageElement extends ilPageContent
         'shibboleth-login-form' => 'shib_login_form',
         'openid-connect-login' => 'openid_connect_login',
         'registration-link' => 'registration_link',
-        'language-selection' => 'language_selection',
```

Comment:
@alex40724 FYI: I had to change this line in your component. I hope this was OK (sine trivial).